waffle_hub/hub/adapter/ultralytics/ultralytics_hub.py

```diff
@@ -260,23 +262,30 @@ def on_train_start(self, cfg: TrainConfig):
 
     def training(self, cfg: TrainConfig, callback: TrainCallback):
 
-        model = YOLO(cfg.pretrained_model, task=self.backend_task_name)
+        code = f"""if __name__ == "__main__":
```

Comment:
Wouldn't it be better to create a separate function? It looks strange as it is.
like as there include " for a string type,

Reply:
I cannot understand what you saying.